hmda/src/main/scala/hmda/validation/filing/ValidationFlow.scala

```diff
@@ -142,7 +142,20 @@ object ValidationFlow {
     errors.map(error => {
       val affectedFields = EditDescriptionLookup.lookupFields(error.editName, period)
       val fieldMap =
-        ListMap(affectedFields.map(field => (field, lar.valueOf(field))): _*)
+        error.editName match {
+          case "V718" =>
+            ListMap(
+              affectedFields.map(
+                field =>
+                  (
+                    field,
+                    "Provided: " + lar.valueOf(field) + ", Expected: " + period.quarter
```

Comment:
@kgudel  tested this out locally and the fields aren't populating with this data.

<img width="1039" alt="Screen Shot 2020-02-06 at 1 24 07 AM" src="https://user-images.githubusercontent.com/44584592/73911802-e3052380-4880-11ea-9e93-e6a46418b109.png">